plugins/thunderbolt/fu-thunderbolt-image.c

```diff
@@ -400,6 +460,8 @@ get_host_locations (guint16 id)
 		{ .offset = 0x136, .len = 1, .description = "Src0", .mask = 0xF0 },
 		{ .offset = 0xB6,  .len = 1, .description = "PA/PB (USB2)", .mask = 0xC0 },
 		{ .offset = 0x5E,  .len = 1, .description = "Aux", .mask = 0x0F },
+		{ .offset = 0x45,  .len = 1, .description = "Flash Size", .mask = 0x07 },
```

Comment:
Interesting.
1. I think we have a similar field in AR too. Worth adding it?
2. I think we haven't added it for AR as it theoretically can be changed (e.g. rev1 expected 8MB flash, rev2 is optimized to require only 4MB).

Reply:
This actually was added after internal review by the Windows/SDK people. I'll make a patch to add the same for AR.